node_modules/oae-core/linkpreview/js/linkpreview.js

```diff
@@ -135,9 +137,19 @@ define(['jquery', 'oae.core', 'jquery.oembed'], function($, oae) {
                     embeddable = false;
                 }
             }
+            // Only embed text, html, images, and types that the browser has plugins to support
+            var type = widgetData.previews.targetType ? widgetData.previews.targetType.split(';')[0] : false;
+            var universalSupport = _.contains(['text/html', 'text/plain', 'image/png', 'image/jpeg', 'image/gif', 'image/svg+xml'], type);
+            var havePlugin = navigator.mimeTypes && navigator.mimeTypes[type] && navigator.mimeTypes[type].enabledPlugin;
+            var supported = widgetData.previews.targetType && (universalSupport || havePlugin);
+
             // The link can be embedded as an iFrame
-            if (embeddable) {
-                oae.api.util.template().render($('#linkpreview-iframe-template', $rootel), {'link': link}, $('#linkpreview-container'), $rootel);
+            if (embeddable && supported) {
+                if (type.substring(0, 6) === 'image/') {
+                    oae.api.util.template().render($('#linkpreview-image-template', $rootel), {'img': link, 'linkProfile': widgetData}, $('#linkpreview-container'), $rootel);
+                } else {
+                    oae.api.util.template().render($('#linkpreview-iframe-template', $rootel), {'link': link}, $('#linkpreview-container'), $rootel);
```

Comment:
This still needs addressing. When a link points to an HTML page that can not be embedded, we should use the existing approach where we show a screenshot of the page with a note above the screenshot. However, when the link points to something we can't take a screenshot of (e.g. file, etc.), we should be using the `Open link` tile approach shown above. 

Currently, when I add a link to a ZIP file (e.g. http://www.colorado.edu/conflict/peace/download/peace_example.ZIP), I see a blank screenshot, rather than the expected `Open link` tile.